src/fairchem/data/oc/utils/geometry.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import annotations
+
+import numpy as np
+
+# Code adapted from https://github.com/henriasv/molecular-builder/tree/master
+
+
+class Geometry:
+    """Base class for geometries.
+
+    :param periodic_boundary_condition: self-explanatory
+    :type periodic_boundary_condition: array_like
+    :param minimum_image_convention: use the minimum image convention for
+                                     bookkeeping how the particles interact
+    :type minimum_image_convention: bool
+    """
+
+    def __init__(
+        self,
+        periodic_boundary_condition=(False, False, False),
+        minimum_image_convention=True,
+    ):
+        self.minimum_image_convention = minimum_image_convention
+        self.periodic_boundary_condition = periodic_boundary_condition
```

Comment:
Are this attributes used somewhere? If not we can just remove the init and have this be a simple mixin class for `PlaneBoundTriclinicGeometry` and `BoxGeometry`. It actually looks like the Geometry parent class is never initialized in the derived classes anyway.